CHANGELOG.md

```diff
@@ -203,26 +56,82 @@ n/a
 - Code quality & style improvements implemented based on ESLINT rules
   - import/no-duplicates
   - prefer-const
   - object shorthand
   - add type for imports
   - Prefer-nullish-coalescing
   - No confusing void expression fixes
+  - ban-types
+  - removed extra semicolons to avoid warnings
+  - no-empty-function
+  - prefer-optional-chain
 - Digital Twin content (pages, overlays, translation content, etc.) removed due to dDTR release and decommissioning of the central DTR
 - Build images also for arm64, in addition to amd64
+- Updated data grid table props to the new prop value
+- Upgraded dependencies to latest version
+- Trivy scan: changed to no failure on high findings, as it should only fail if there is an error/misconfiguration
 
 ### Bugfix
 
 - "Dataspace" Introduction page background color of last section changed
+- App Release Process
+  - fixed double loading of images for app page
+  - fixed validation for contact in app page
+  - fixed salesManagerId(null) error
+  - fixed error message deletion upon deleting of csv file in technical integration
+  - fixed consent check removal issue on save
+- Service Release Process
+  - fixed consent check removal issue on save
+- App Overview
+  - fixed image display in app overview cards
+- IDP Configuration
+  - added load element for IDP list display function
+- Home
+  - Fixed image display for 'my business applications'
+  - Fixed white screen due to 'my business applications'
+- Partner Network - update search query parameter name to legalName
+- Technical User Management
+  - technical user creation overlay: type subheading typo fixed
+  - overview page: displaying duplicates after adding/creating new technical users fixed
+  - user detail page: removed copy icon from connector link
+- Admin Business Partner Registration Request Screen - Detail Overlay country fixed to display country code received via backend api
+- Invite Business Partner
+  - fixed auto page refresh/update when inviting via the invite overlay a new company
+  - fix invite overlay loader position
+- Service Subscription
+  - activation overlay field names updated
+  - field value mapping to backend api response properties updated/changed
+  - copy feature/icon added for the technical user secret
+  - header description updated
+  - sorting element behavior fixed in case the user sorts multiple times with the same sorting option
+- Vulnerability from dependency
+  - upgrade axios dependency to v1.6.1
+  - Set resolution for @babel/traverse (CVE-2023-45133)
+- Legal-notice for about page
+  - Re-introduce placeholder values to be replaced during the build process
+- Others
+  - Fixed broken KeyValueView/index.tsx for undefined object attributes
 
 ### Known Knowns
 
-- App Release Process - Step 2 image gets multiple times uploaded if the user uploads the images single instead of dropping all images jointly for upload
+- App Release Process
+  - help link hyperlink incorrectly set
```

Comment:
this is fixed for all help links, beside the specific user role upload help link - this one is still open